docker/Dockerfile

```diff
@@ -33,8 +44,8 @@ RUN cd /opt/lmdeploy &&\
     make -j$(nproc) && make install &&\
     cd .. &&\
     python3 -m pip install -e . &&\
+    python3 -m pip install --no-cache-dir triton==2.1.0 &&\
```

Comment:
why triton 2.1?

Reply:
the performance of pytorch engine is better on triton==2.1.0.

Reply:
Is it compatible with torch2.3?

Reply:
yes.

Reply:
@grimoire is it?

Reply:
Performance of 2.2 is accepable, 2.3 is slow.

Reply:
If triton version doesn't depend on torch, can we set "triton>=2.1.0,<=2.2" in runtime.txt?

Reply:
> If triton version doesn't depend on torch

torch do

Reply:
ok